nexus/types/src/external_api/params.rs

```diff
@@ -1172,23 +1172,32 @@ pub struct InstanceCreate {
     #[serde(default)]
     pub external_ips: Vec<ExternalIpCreate>,
 
-    /// The disks to be created or attached for this instance.
+    /// A list of disks to be attached to the instance.
+    ///
+    /// Each disk can either be attached if it already exists or created along
+    /// with the instance.
+    ///
+    /// The ordering of the disks does not guarantee the boot order for the
+    /// instance. Use the boot_disk attribute to configure the disk the instance
+    /// should boot from.
```

Comment:
```suggestion
    /// The order of this list does not guarantee a boot order for the
    /// instance. Use the boot_disk attribute to specify a boot disk.
```

Reply:
I used "attribute" since the public documentation for this is presented in the context of a JSON request body. I also don't have a better word. I updated the wording as suggested.